views/includes/head.hbs

```diff
@@ -14,5 +14,183 @@
 <!-- Start of head include. -->
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
+
+<style>
+html, body {
+  font-family: "Helvetica", "Arial", sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  margin: 0;
+  height: 100%;
+  width: 100%;
+}
+body {
+  background: #F5F5F5;
+}
+.container {
+    padding-right: 15px;
+    padding-left: 15px;
+    margin-right: auto;
+    margin-left: auto
+}
+.navbar {
+  height: 80px;
+  background: #424242;
+  display: flex;
+  flex-direction: column;
+}
+.navbar-title {
+  color: white;
+  font-size: 30px;
+  text-align: center;
+  margin-top: 10px;
+}
+.navbar-subtitle {
+  color: white;
+  font-size: 18px;
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+.section-title {
+  height: 30px;
+  color: white;
+  font-size: 18px;
+  background: #757575;
+  padding-top: 5px;
+  padding-left: 15px;
+  margin-bottom: 15px;
+}
+
+.navbar a {
+  color: white;
+  text-decoration: none;
+}
+
+.pwa-buttom {
+  font-size: 14px;
+  color: white;
+  width: 100px;
+  height: 30px;
+  text-align: center;
+  line-height: 30px;
+  margin-bottom: 15px;
+  display: inline-block;
+  border:none;
+}
+
+.green {
+  background: #33691E;
+}
+
+.blue {
+  background: #0D47A1;
+}
+
+.red {
+  background: #B71C1C;
+}
+
+.pwa-buttom  a {
+  color: white;
+  text-decoration: none;
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.media {
+  border: 2px solid lightgray;
+  padding: 8px 4px 4px 8px;
+  text-overflow: ellipsis;
```

Comment:
Can remove text-overflow: ellipsis for now, as it's not working. Maybe adding "overflow: hidden;" so it won't show outside the box and ellipsizing the headers and texts elements.